### PR TITLE
Update PPM Panel Homepage Timeline From Image to CSS

### PR DESCRIPTION
**Pull request by ralren**
## Description

The PPM Panel homepage has the timeline sourced as an image. As HHG already implements the timeline with CSS and there are plans to add more phases to the PPM timeline, it had been decided to replace the images with CSS.

## Reviewer Notes
I'm a bit iffy on the timeline implementation for `DraftMoveSummary`. The image for it seems to default to having the first two stages completed with no conditionals so I just automatically marked those two stages as completed.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:
```sh
make db_dev_e2e_populate
make server_run
make client_run
```
#### Tests
```
make client_test
```

## Code Review Verification Steps
* [x] User facing changes have been reviewed by design.
* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/161963161) for this change

## Screenshots
Screen Shot 2019-03-26 at 3 30 17 PM
Screen Shot 2019-03-26 at 3 30 47 PM

### CODE REVIEW

File: src/scenes/Landing/StatusTimeline.jsx [hunk 1]
```diff
@@ -52,68 +81,84 @@ export class StatusTimelineContainer extends PureComponent {
...
+      { name: 'Packed', code: 'PACKED', dates: 'pack' },
+      { name: 'Loaded', code: 'LOADED', dates: 'pickup' },
+      { name: 'In transit', code: 'IN_TRANSIT', dates: 'transit' },
+      { name: 'Delivered', code: 'DELIVERED', dates: 'delivery' },
+    ];
+    const statusBlocks = [];
     const formatType = 'condensed';
 
-    const markedStatuses = this.determineCompletedAndCurrentStatuses(this.props.shipment);
+    let statuses = HHGSTATUSES;
```
Comment (mkrump) [inline src/scenes/Landing/StatusTimeline.jsx]: The if else blocks for both `statuses` and `markedStatuses` makes it feel like the individual menu items should be their own components or classes. I'm not sure what the most idiomatic javascript / react way of doing that is.  But in python or similar i'd just pull out classes and rely on ducktyping for the one common method. Something like this maybe https://gist.github.com/mkrump/8a2c1e6c2e1fd921e86fa09bffab3397? Would be curious to hear @jim, @Ronolibert or others opinions.

Comment (mkrump) [reply src/scenes/Landing/StatusTimeline.jsx]: At a minimum it feels like the determination of the statuses and marked statuses could be pulled into a common method outside of render https://gist.github.com/mkrump/5437e001b76e2317a93f4ac73ac267b5 sort of thing?

File: src/scenes/Landing/StatusTimeline.jsx [hunk 2]
```diff
@@ -13,26 +13,55 @@ export class StatusTimelineContainer extends PureComponent {
...
     if (!statuses.includes(statusToCheck)) {
       return false;
     }
     return indexOf(statuses, statusToCheck) === statuses.length - 1;
   }
 
-  determineCompletedAndCurrentStatuses(shipment) {
-    const today = moment();
+  determineCompletedAndCurrentProfileStatuses() {
+    return ['PROFILE', 'ORDERS'];
```
Comment (mkrump) [inline src/scenes/Landing/StatusTimeline.jsx]: Like you said in comments above, it seems like there should be some additional logic here for the profile status. Agree would be good to clarify this, before finalizing.

File: src/scenes/Landing/StatusTimeline.jsx (hunk 1 above)
Comment (ralren) [reply src/scenes/Landing/StatusTimeline.jsx]: I quite like the implementation you provided. I would like to hear if other folks have implementations they would like to suggest first.

Comment (ralren) [reply src/scenes/Landing/StatusTimeline.jsx]: @mkrump, I went ahead and used your implementation if you want to take another look.

**Pull request merged**